pkg/cmd/infra/router/template.go

```diff
@@ -87,12 +95,36 @@ func reloadInterval() time.Duration {
 	interval := util.Env("RELOAD_INTERVAL", fmt.Sprintf("%vs", defaultReloadInterval))
 	value, err := time.ParseDuration(interval)
 	if err != nil {
-		glog.Warningf("Invalid RELOAD_INTERVAL %q, using default value %v ...", interval, defaultReloadInterval)
+		glog.Warningf("Invalid RELOAD_INTERVAL %q, using default value %vs ...", interval, defaultReloadInterval)
 		value = time.Duration(defaultReloadInterval * time.Second)
 	}
 	return value
 }
 
+// reloadGap returns how often to wait between a reload end and the start of the next reload.
+// The gap value is based on an environment variable or the default.
+func reloadGap() time.Duration {
+	gap := util.Env("RELOAD_GAP", fmt.Sprintf("%vms", defaultReloadGap))
+	value, err := time.ParseDuration(gap)
+	if err != nil {
+		glog.Warningf("Invalid RELOAD_GAP %q, using default value %vms ...", gap, defaultReloadGap)
+		value = time.Duration(defaultReloadGap * time.Millisecond)
+	}
+	return value
+}
+
+// reloadEventWait returns how often to wait after getting the first event after a reload before triggering the next reload.
+// The event wait value is based on an environment variable or the default.
+func reloadEventWait() time.Duration {
+	eventWait := util.Env("RELOAD_EVENT_WAIT", fmt.Sprintf("%vms", defaultReloadEventWait))
+	value, err := time.ParseDuration(eventWait)
+	if err != nil {
+		glog.Warningf("Invalid RELOAD_EVENT_WAIT %q, using default value %vms ...", eventWait, defaultReloadEventWait)
+		value = time.Duration(defaultReloadEventWait * time.Millisecond)
+	}
+	return value
+}
```

Comment:
Kinda seems like these could be one function that takes the env var name and the default interval as arguments.

Reply:
Done